src/Compilers/CSharp/Portable/Operations/CSharpOperationFactory.cs

```diff
@@ -1771,9 +1772,11 @@ private IUsingOperation CreateBoundUsingStatementOperation(BoundUsingStatement b
             ImmutableArray<ILocalSymbol> locals = boundUsingStatement.Locals.GetPublicSymbols();
             bool isAsynchronous = boundUsingStatement.AwaitOpt != null;
             IMethodSymbol? disposeMethod = boundUsingStatement.DisposeMethodOpt.GetPublicSymbol();
+            var receiver = boundUsingStatement.ExpressionOpt ?? boundUsingStatement.DeclarationsOpt?.LocalDeclarations[0].DeclaredTypeOpt;
```

Comment:
>var receiver = boundUsingStatement.ExpressionOpt ?? boundUsingStatement.DeclarationsOpt?.LocalDeclarations[0].DeclaredTypeOpt; [](start = 12, length = 126)

If the only thing we need a type, consider getting it right here. Otherwise, it is not clear why we are looking only at the first local declaration